Enterprise-CMCS/macpro-mdct-mfp
# Disable Closeout Button

### Description

Task
1) Disable closeout button until previous fields are filled
2) Update closeout text

### Related ticket(s)


CMDCT-201

---
### How to test

1) In local, find script **EntityRow.tsx**  and comment out line [107] [disabled={entityStatus === "disabled"}] to enable access to the close out entity
2) Sign into MFP as a state user
3) Create a new WP
4) Add a new initiative in State and Territory-Specific Initiatives and press the Edit button 
5) Edit  Close-out information
6) The [close out initiative] button should now disable or enable based on whether the fields above it are filled out

Note: The form doesn't actually close out atm.

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
